rust/lanzaboote/src/pe_section.rs

```diff
@@ -1,3 +1,9 @@
+// Clippy doesn't like the lifetimes, but rustc wants them. 🤷
+#![allow(clippy::needless_lifetimes)]
```

Comment:
could that be those lifetimes… are not necessary :> ? or is there any upstream bug?

Reply:
Might be an issue in nightly, but it didn't feel important enough to dig deeper. Feel free to try to fix it. :)